frontend/src/pages/results.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useRef, useState } from "react";
+import Chart from "chart.js";
+import Button from "react-bootstrap/Button";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Grid from "@material-ui/core/grid";
+import Stepper from "./stepper.js";
+import Radar from "./chart/chart";
+import EMenu from "./notesMenu.js";
+import { Typography, Box } from "@material-ui/core";
+
+function Results() {
+  let Summary_Value = 2.03;
+  let Coverage = { Safety: 0.5, Salary: 0.667, Reputation: 1.0, Privacy: 0.8 };
+  return (
+    <div>
+      <Grid direction="row" justify="center" alignItems="center">
+        <Typography variant="h4" align="center" gutterBottom="true">
+          Coverage Of Issues
+        </Typography>
+      </Grid>
+      <Grid container direction="row" justify="space-between">
+        <Grid item style={{ marginLeft: "2rem", marginBottom: "1rem" }}>
+          <Button variant="light">Back</Button>
+        </Grid>
+        <Grid item style={{ marginRight: "2rem", marginBottom: "1rem" }}>
+          <Button variant="primary">Next</Button>
+        </Grid>
+      </Grid>
+      <Grid container spacing={2}>
+        <Grid item lg={3}>
```

Comment:
For the item sizes (lg={3}), you should play around with what needs to happen to make it mobile responsive. If you emulate a phone what happens? lg={3} means it takes up 3/12 spots on screens 'large' and up. Should it be bigger for xs and up? (for example you can write XS={12} sm={6} lg={3} and extra small will be 12/12, small and medium will be 6/12 and large and extra large will be 3/12

Reply:
Added box dimensions for medium and small screen sizes, however, a different UI design should be implemented later. Made a useWindowSize() in gatheredList.js that can later be used to make the entire UI screen size responsive.